duckduckgo/iOS
# Add site rating to content blocker popover

Reviewer: Chris
Asana: https://app.asana.com/0/361428290920652/410205284101812

**Description**:
Adds the site rating to the content blocker popover
- creates a new Site Rating view so that code is not duplicated between the omnibar and popover
- moves site blocking stats from ContentBlocker to SiteRating where site level scope is more appropriate

**Steps to test this PR**:
1. Browse to a page
2. Tap on the site rating
3. Note that the content blocker popover opens and contains a site rating entry

###### Reviewer Checklist:


###### PR DRI Checklist:
